internal/documentdb/genwrap/convert.go

```diff
@@ -289,3 +284,73 @@ func handleFunctionOverloading(f *convertedRoutine) {
 		f.Name = funcName
 	}
 }
+
+// groupBySpecificName groups rows by specific_name.
+func groupBySpecificName(rows []map[string]any) map[string][]map[string]any {
```

Comment:
I wonder if it makes sense to make all those free-standing functions methods on a Converter type just to group them together

Reply:
It might make it cleaner visually.

Reply:
Actually this also allows (unnecessary) conversion done in generate.go. So it added value.